models/points.go

```diff
@@ -741,6 +741,16 @@ func scanLine(buf []byte, i int) (int, []byte) {
 	start := i
 	quoted := false
 	fields := false
+	bufLength := len(buf)
+
+	for {
+		if bufLength > start && buf[start] == 0 {
```

Comment:
What if I had filled the buffer with some other non-zero value, which was not a valid character? Is that possible?